lib/modules/datasource/pypi/index.ts

```diff
@@ -34,45 +34,51 @@ export class PypiDatasource extends Datasource {
     packageName,
     registryUrl,
   }: GetReleasesConfig): Promise<ReleaseResult | null> {
-    let dependency: ReleaseResult | null = null;
+    const dependency: ReleaseResult = { releases: [] };
     // TODO: null check (#22198)
-    const hostUrl = ensureTrailingSlash(
-      registryUrl!.replace('https://pypi.org/simple', 'https://pypi.org/pypi'),
-    );
+    let hostUrl = ensureTrailingSlash(registryUrl!);
```

Comment:
Create separate variables for the URLs than rewriting the variable

Reply:
should be fixed now